brave_paks.gni

```diff
@@ -5,7 +5,7 @@
 import("//brave/components/brave_ads/browser/buildflags/buildflags.gni")
 import("//brave/components/brave_rewards/browser/buildflags/buildflags.gni")
 import("//brave/components/brave_webtorrent/browser/buildflags/buildflags.gni")
-import("//brave/components/speedreader/buildflags.gni")
```

Comment:
Can't we just drop (vs fix) this include? I don't see it used anywhere in this file...